content/1.7/back-office/customers/customers/view-customer.md

```diff
@@ -0,0 +1,82 @@
+
+Below there are 12 sections with infomration about the customer.
+
+### Columns
+
+**Customer name, number and email** - on the right of this section is an edit button, when pressed redirects to [specs of the add edit customer]({{<ref "add-edit-customer.md">}} "Add edit customer")
+
+Below the following information about the customer is displayed:
+
+ - Social title: if the title is not input, just shows "Unknown"
+ - Age: if the title is not input, just shows "Unknown"
+ - Registration date
+ - Last visit
+ - Shop: shows the shop that the customer is registered on
+ - Language
+ - Registrations: shows the possible registrations, and in an red square if the customer is not registered there
+ - Latest Update
+ - Status: shows if the customer is active
+
+**Orders** - on the right of the title the number of orders is displayed. Below the number of valid, invalid orders and the amount (in the default currency of the shop) is displayed.
+All the orders are listed below, and the details of the orders are displayed:
+ - ID
+ - Date
+ - Payment
+ - Status
+ - Products
+ - Total spent
+ - Actions (when pressed, redirects to the order page).
+
+**Carts** - Displays the number of carts by the customer
+All the carts for the customer are displayed:
+ - ID
+ - Date
+ - Carrier
+ - Total
+ - Actions (when pressed, redirects to the cart page).
+
+**Viewed products** - Displays the number of viewed products
+Below all the products are listed:
+ - ID
+ - Name
+
+**Add a private note** - a help text in the box displays: "This note will be displayed to all employees but not to customers.".
+Below a text box is displayed that shows: "Add a note on this customer. It will only be visible to you.". When pressed, it's possible to add text.
+Below there is a "save" button which saves the input text.
+
+**Messages** - Displays the messages and their count from the customer, if there are no messages, it will show: "customer name has never contacted you".
+
+**Vouchers** - Displays the vouchers and their count that the customer has, if there are no vouchers, it will show: "customer name has no discount vouchers".
+
+**Last emails** - Displays the last emails from the customer and their count.
+Below the following information is shown:
+ - Date
+ - Language
+ - Subject
+ - Template
+
+**Last connections** - Displays the last connections and their count.
+Below the following information is shown:
+ - ID
+ - Date
+ - Pages viewed
+ - Total time
+ - Origin
+ - IP address
+
+**Groups** - Displays the groups and their count, also an edit button is displayed, when pressed edirects to [specs of the add edit customer]({{<ref "add-edit-customer.md">}} "Add edit customer")
+Below the following information is showed:
+ - ID
+ - Name
+
+**Addresses** - Displays the addresses and their count. Has an "add" button, when pressed, redirects to the customer addresses page.
+
+**Wishlist** - Displays the count of the items added to the wishlist by the customer.
```

Comment:
Proposed wording :

#### Wishlist
The panel header displays the title **Wishlist** and the number of items in the wishlists in a blue badge.